pep-0615.rst

```diff
@@ -190,9 +190,9 @@ of the interpreter, it must not invalidate any caches or modify any
 existing ``ZoneInfo`` objects. Newly constructed ``ZoneInfo`` objects, however,
 should come from the updated data source.
 
-This means that the point at which a ``ZoneInfo`` file is updated depends
+This means that the point at which a ``ZoneInfo`` object is updated depends
```

Comment:
I agree that `file` is wrong here, but I think `object` is not right either.

I think maybe this is better:

> "This means that the point at which the data source is updated for new invocations of the ``ZoneInfo`` constructor depends primarily on the semantics of the caching behavior.

Reply:
TBH I can't really tell what it's trying to say. In plainer language should it just be this?

> Therefore, whether updates can be seen depends primarily on the cache behaviour.

Reply:
I don't think so. The point to convey here is that for something like:

```python
while True:
    print(datetime(2025, 11, 1, 13, tzinfo-ZoneInfo("America/New_York")))
```

It if you update `tzdata` during the run of that program and there are no other living references to `ZoneInfo("America/New_York")`, it is implementation-defined whether the output of that program would change if the "America/New_York" TZif file changes on disk. Compared to that:

```python
a = ZoneInfo("America/New_York")
while True:
    print(datetime(2025, 11, 1, 13, tzinfo-ZoneInfo("America/New_York")))
```

The result is guaranteed here _not_ to change.

I don't think that really qualifies as "whether updates can be seen". I think / hope the surrounding text makes this more clear.